hikari/impl/bot.py

```diff
@@ -332,16 +307,16 @@ def __init__(
 
         # We populate these on startup instead, as we need to possibly make some
         # HTTP requests to determine what to put in this mapping.
-        self._shards: typing.Dict[int, shard.GatewayShard] = {}
+        self._shards: typing.Dict[int, gateway_shard.GatewayShard] = {}
         self.shards: typing.Mapping[int, gateway_shard.GatewayShard] = types.MappingProxyType(self._shards)
 
     @property
     def cache(self) -> cache_.Cache:
         return self._cache
 
-    @property
-    def chunker(self) -> chunker_.GuildChunker:
-        return self._chunker
+    # @property
+    # def chunker(self) -> guild_chunker.GuildChunker:
```

Comment:
what're we doing with the chunker?

Reply:
Since the chunker relies on the cache, but currently disabling the cache will not trigger the cache anywhere, im going to be rewriting it to work with both. Since this pr is already really big, it will be done before the feature branch is merged into master